bitcoin/src/parse.rs

```diff
@@ -90,8 +90,19 @@ pub(crate) fn int<T: Integer, S: AsRef<str> + Into<String>>(s: S) -> Result<T, P
     })
 }
 
+/// Strips the hex prefix off `s` if one is present.
+pub fn strip_prefix<S: AsRef<str> + Into<String>>(s: &S) -> &str {
```

Comment:
This is totally out of context and the doc doesn't help either. I'd call it either `strip_hex_prefix` or `strip_0x_prefix`. Also I find it strange to make this `pub` and `Into<String>` is unused - better to just accept `&str`.

Reply:
Yes to all suggestions, also I changed it to `pub(crate)`, at least until #2498 is done.